Parser/OperatingSystem.php

```diff
@@ -280,7 +284,7 @@ class OperatingSystem extends AbstractParser
      * @var array
      */
     private $lineageOsVersionMapping = [
-        '14'    => '21.0',
+        '14'    => '21',
```

Comment:
Is there a specific reason for removing the `.0`?

Reply:
> Is there a specific reason for removing the `.0`?

https://lineageos.org/Changelog-28/
_As Android has moved onto the quarterly maintenance release model, this release will be “LineageOS 21”, not 21.0 or 21.1 - though worry not - we are based on the latest and greatest Android 14 version, QPR1._